apps/passport-server/test/devconnect.spec.ts

```diff
@@ -1654,12 +1655,15 @@ describe("devconnect functionality", function () {
   step(
     "user should be able to be issued some PCDs from the server",
     async function () {
+      MockDate.set(new Date());
```

Comment:
This is to lock the system date during the test, so that we don't accidentially get an expired credential when the hour rolls over.